infrastructure/cdn-in-a-box/enroller/enroller.go

```diff
@@ -560,9 +560,12 @@ func enrollProfile(toSession *session, r io.Reader) error {
 			}
 		}
 		profiles, _, err = toSession.GetProfileByName(profile.Name)
-		if err != nil || len(profiles) == 0 {
+		if err != nil {
 			log.Infof("error getting profile ID from %+v: %s\n", profile, err.Error())
 		}
+		if len(profiles) == 0 {
+			log.Infof("no results returned for getting profile ID from %+v", profile)
+		}
 		profile = profiles[0]
```

Comment:
This could panic, right? This seems like a fatal error anyways, but maybe the enroller should exit gracefully instead.

Reply:
I agree, I didn't make the change because for all the other enroller methods, they only log not panic. I can change all of them but I figured something like that is better in it's own PR due to the potential side affects.